packages/hardware-trezor/src/transformers/keyPaths.ts

```diff
@@ -16,6 +16,19 @@ export const paymentKeyPathFromGroupedAddress = (address: GroupedAddress): BIP32
   address.index
 ];
 
+export const stakeKeyPathFromGroupedAddress = (address: GroupedAddress): BIP32Path | null => {
+  if (address.stakeKeyDerivationPath) {
+    return [
+      util.harden(CardanoKeyConst.PURPOSE),
+      util.harden(CardanoKeyConst.COIN_TYPE),
+      util.harden(address.accountIndex),
+      address.stakeKeyDerivationPath.role,
+      address.stakeKeyDerivationPath.index
+    ];
+  }
+  return null;
```

Comment:
this could be simplified to a one-liner:
```ts
export const stakeKeyPathFromGroupedAddress = (address: GroupedAddress): BIP32Path | null => 
  address.stakeKeyDerivationPath ? [
      util.harden(CardanoKeyConst.PURPOSE),
      util.harden(CardanoKeyConst.COIN_TYPE),
      util.harden(address.accountIndex),
      address.stakeKeyDerivationPath.role,
      address.stakeKeyDerivationPath.index
    ] : null;
```

Reply:
Done in [f3c53ef](https://github.com/input-output-hk/cardano-js-sdk/pull/793/commits/f3c53ef6517130a3c1a8525a2b6cac4961be9f01)